Algorithm-Boo-who.md

```diff
@@ -1,6 +1,8 @@
+# Algorithm Boo who
+
 ![](http://i.imgur.com/CgROCEF.jpg)
 
-# Problem Explanation:
+## Problem Explanation:

```

Comment:
The Explanation should appear in chat at the first call. With `h2`, it won't appear.